packages/tasks/src/model-libraries.ts

```diff
@@ -174,6 +174,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		/// diffusers has its own more complex "countDownloads" query
 	},
+	diffusionkit: {
```

Comment:
Would be nice to have some models listed in https://huggingface.co/models?other=diffusionkit before merging this PR. You can do that by tagging models as `diffusionkit` in the model card metadata.

I opened a first PR to do that: https://huggingface.co/argmaxinc/mlx-FLUX.1-schnell/discussions/2

Reply:
Merged 👍

Reply:
I noticed that the SD3 models have the `DiffusionKit` tag: https://huggingface.co/models?other=DiffusionKit. The correct tag should be `diffusionkit` (lowercased). Make sure to update them to benefit from this PR.

(the correct key is the key defined here, so `diffusionkit` in this case. Below we are defining `prettyLabel` and `repoName` but that's just for aesthetic in the UI)

Reply:
We generated these models with diffusionkit much before we had an MLX implementation, and in fact don't yet have a public CoreML inference implementation of DiffusionKit (in the works). These models are actually more directly usable by the huggingface [swift-coreml-diffusers](https://github.com/huggingface/swift-coreml-diffusers) app, although they were generated with diffusionkit. So in this case we might want to just remove the lowercase `diffusionkit` tag? @pcuenca what do you think about a local-app deeplink for Diffusers.app? We had discussed adding arbitrary repos in the past.

Reply:
Updated our tags and model cards: https://huggingface.co/models?other=diffusionkit